packages/components/molecules/f-alert/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v6.0.0
+------------------------------
+*Jun 23, 2022*
```

Comment:
Please can you change all the `Jun`s to `June`s? 😄

Reply:
🤦🏼